docs: Remove skipMeta and skipEventId parameters (#126)
dhis2/dhis2-releases: releases/2.41/README.md

effect on the response

* `/tracker/trackedEntities?attachment`
* `/tracker/events?attachment`

> * `/tracker/trackedEntities?skipMeta=true`

Reviewer: ```suggestion
* `/tracker/trackedEntities?skipMeta`
```

> * `/tracker/events?skipMeta=true`

Reviewer: ```suggestion
* `/tracker/events?skipMeta`
```

> Parameter `skipEventId` in `GET /tracker/events` endpoint was removed as include or exclude fields from the JSON response can be achieved by using the fields query parameter.

Reviewer: I think skipMeta and skipEventId have been removed for that reason 🤔 

```suggestion
Parameters `skipEventId` in `GET /tracker/events` endpoint was removed as including or excluding fields from the JSON response can be achieved by using the `fields` query parameter.
```
